### ISSUE TITLE
Disable Sensorless

### ISSUE THREAD

**Issue by snow79** [makerbase-mks/MKS-Robin-Nano-V3.X#109]
Hello Guys
i have MKS robin nano v3.1 and driver TMC2209 in uart mode.
I like to use the standard sensor and disable the sensorless, but i don't find how i can do it.
Please some one can explay how i can do it?
Hardware i have try with end without jumper on diag x, diag y, diag z but nothing change;
on firmware marlin i dont' have enable //#define SENSORLESS_HOMING
the printer don't use sensor and don't stop :(

**Comment by MKS-hosiey** [comment]
Is enstop enabled correctly? Is your machine a normal model or a corexy?

**Comment by AngelBautista** [comment]
Hello,
I have a MKS robin nano v3.1 and TCM2209 driver, I would like you to help me configure it, it is a home printer, eventually I want to evolve it to x,y. I am a complete novice and I would like to know if by default the sensorless is enabled in the firmware. Thank you very much for your help.

**Comment by snow79** [comment]
Step 1 you like to use or not sensorless?If yes you have to enable on firmware and but jumper diag on.If no you have to set on firmware X y limit and remove jumper.For any help I'm hereSent from Android deviceIl 17 mag 2022 20:22, AngelBautista ***@***.***> ha scritto:
Hello,
I have a MKS robin nano v3.1 and TCM2209 driver, I would like you to help me configure it, it is a home printer, eventually I want to evolve it to x,y. I am a complete novice and I would like to know if by default the sensorless is enabled in the firmware. Thank you very much for your help.

**Comment by AngelBautista** [comment]
Yes, I would like to use the sensorless option.  which jumper should I use?

![image](https://user-images.githubusercontent.com/16033696/169693134-262f8e0d-792a-4c26-98fa-8aa4f9a4bf93.png)

Thank you very much

**Comment by nfalco79** [comment]
the DIAG jumper are those marked as number 3. Each row enable a sensorless homing for one single axis
